homeassistant/components/synology_dsm/camera.py

```diff
@@ -94,7 +98,27 @@ def camera_image(self) -> bytes:
         """Return bytes of camera image."""
         if not self.available:
             return None
-        return self._api.surveillance_station.get_camera_image(self._camera.id)
+        try:
+            return self._api.surveillance_station.get_camera_image(self._camera.id)
+        except (SynologyDSMAPIErrorException) as err:
+            _LOGGER.debug(
+                "SynoDSMCamera.camera_image() - Exception:%s",
+                err,
+            )
+            return None
+
+    @property
+    def should_poll(self) -> bool:
+        """Return the polling state."""
+        return True
+
+    async def async_update(self):
+        """Update camera state."""
+        await self.hass.async_add_executor_job(self._api.surveillance_station.update)
```

Comment:
This is going to update every entity another time for each camera that is added.  We should use DataUpdateCoordinator

Reply:
Thanks @bdraco for the hint with DataUpdateCoordinator 👍